tracer/src/Datadog.Trace/Tagging/AzureServiceBusTags.cs

```diff
@@ -17,6 +17,9 @@ internal partial class AzureServiceBusTags : OpenTelemetryTags
     {
         private string _spanKind;
 
+        [Tag(Trace.Tags.InstrumentationName)]
+        public string InstrumentationName { get; set; }
```

Comment:
Just to ensure it is always set (you'll need to update instances where it was being set explicitly though)

```suggestion
        public string InstrumentationName => nameof(IntegrationId.AzureServiceBus);
```